clients/hadoopfs/src/test/java/io/lakefs/contract/TestLakeFSContractRename.java

```diff
@@ -1,12 +1,10 @@
 package io.lakefs.contract;
 
 import org.apache.hadoop.conf.Configuration;
-import org.apache.hadoop.fs.contract.AbstractContractRenameTest;
-import org.apache.hadoop.fs.contract.AbstractFSContract;
 import org.apache.hadoop.fs.FileSystem;
 import org.apache.hadoop.fs.Path;
-import org.apache.hadoop.fs.contract.ContractTestUtils;
-import org.junit.Test;
+import org.apache.hadoop.fs.contract.AbstractContractRenameTest;
+import org.apache.hadoop.fs.contract.AbstractFSContract;
```

Comment:
Why is this import now needed?

Reply:
It wasn't added - just got to the right place after sort.